src/rules/maxLineLengthRule.ts

```diff
@@ -29,12 +36,24 @@ export class Rule extends Lint.Rules.AbstractRule {
             Limiting the length of a line of code improves code readability.
             It also makes comparing code side-by-side easier and improves compatibility with
             various editors, IDEs, and diff viewers.`,
-        optionsDescription: "An integer indicating the max length of lines.",
+        optionsDescription: Lint.Utils.dedent`
+            An integer indicating the max length of lines. Additionally, the \`\"ignore-urls\"\` option
+            can be passed to ignore lines with long URLs.
+            `,
         options: {
-            type: "number",
-            minimum: "1",
+            type: "array",
+            items: [
+                {
+                    type: "number",
+                    minimum: "1",
+                },
+                {
+                    type: "string",
+                    enum: [OPTION_IGNORE_URL],
+                },
+            ],
         },
-        optionExamples: [[true, 120]],
+        optionExamples: [[true, 120], [true, 140, "ignore-urls"]],
```

Comment:
Should be either a number or an object with a `limit` property, see https://github.com/palantir/tslint/pull/3099#discussion_r131516378